lib/adbc_driver.ex

```diff
@@ -97,6 +97,28 @@ defmodule Adbc.Driver do
         url:
           "https://github.com/apache/arrow-adbc/releases/download/apache-arrow-adbc-0.10.0/adbc_driver_snowflake-0.10.0-py3-none-win_amd64.whl"
       }
+    },
+    duckdb: %{
```

Comment:
We should not change this module attribute because it is auto generated. I have pushed a commit to main that splits the user defined variables from the generated ones. If you rebase you should be good!